src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,454 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Customize web media notifications and respond to media related events with the new Media Session API.
+
+{# wf_updated_on: 2017-02-06 #}
+{# wf_published_on: 2017-02-06 #}
+{# wf_tags: news,chrome57,media,notifications,play #}
+{# wf_featured_image: /web/updates/images/2017/02/tldr.png #}
+{# wf_featured_snippet: Finally! We can customize web media notifications (title, artist, album name, artwork) and respond to media related events such as seeking or track changing with the new Media Session API. #}
+
+# Customize Media Notifications & Handle Playlists {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+With the brand new [Media Session API], you can now **customize media
+notifications** by providing metadata information for the media your web app is
+playing. It also allows you to **handle media related events** such as seeking
+or track changing which may come from notifications or media keys. Excited? Try
+out the official [Media Session sample]!
+
+The Media Session API is supported in Chrome 57 (beta in February 2017, stable
+in March 2017).
+
+<figure>
+  <img src="/web/updates/images/2017/02/tldr.png"
+    alt="Media Session TL;DR;"/>
+</figure>
+
+## Gimme What I Want
```

Comment:
All titles level 2 and lower should be sentence case. Only the article title should be title case. Please change throughout.